dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/README.md

```diff
@@ -47,6 +49,10 @@ The higher the risk, the more extensive the informed consent procedures must be,
 - Signed consent form 
 - Consent specific to the recording medium (e.g., videotape)
 
+## Surveying minors
+When surveying minors (aged 8 to 17), an assent needs to be obtained from a parent or guardian unless an exemption is provided by the IRB. This [template](https://couhes.mit.edu/sites/default/files/documents/Assent-Form-2019-01-22.doc) can be used to develop an assent form. 
+
+## Resources
```

Comment:
should probably add DPO sample approved consent form to this list of resources: https://worldbankgroup.sharepoint.com/sites/wbsites/WBDP/Sitepages/Detail.aspx/Documents/mode=view?_Id=45&SiteURL=/sites/wbsites/wbdp